src/main/java/com/superzanti/serversync/client/Mode2Sync.java

```diff
@@ -22,28 +27,46 @@ public static Mode2Sync forServer(Server server) {
     public void run() {
         FileManifest manifest = server.fetchManifest();
 
-        manifest.entries
-            .forEach(entry -> {
-                Path file = entry.resolvePath();
-                Logger.debug(String.format("Starting check for file: %s", file));
-                if (!entry.redirectTo.equals("")) {
-                    Logger.debug(String.format(
+        Gui_JavaFX.getStackMainPane().getPaneSync().getObservMods().clear();
```

Comment:
The sync will need to be agnositc of the GUI, I'll probably need to change this to keep the command line intact.